### PR TITLE
New endpoints for VUI web API

### PR DESCRIPTION
**Pull request by davidraker**
# Description

Added enabled, running, status, and tag endpoints.  Added documentation for these and improved installation documentation, including for setting up the web-users.json file.

Fixes # (issue)
VOLTTRON/volttron#2862
VOLTTRON/volttron#2865
VOLTTRON/volttron-web#2
VOLTTRON/volttron-web#3
VOLTTRON/volttron-web#4
VOLTTRON/volttron-web#5
VOLTTRON/volttron-web#6
VOLTTRON/volttron-web#7
VOLTTRON/volttron-web#18
VOLTTRON/volttron-web#21
VOLTTRON/volttron-web#22
VOLTTRON/volttron-web#23
VOLTTRON/volttron-web#28
VOLTTRON/volttron-web#29
VOLTTRON/volttron-web#30
VOLTTRON/volttron-web#31

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

Tests for all new functionality are included in test_vui_endpoints.py.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by craig8 — COMMENTED** [review]
Not done yet, but so far so good.

File: docs/source/platform-features/web-api/introduction.rst [hunk 1]
```diff
@@ -17,6 +17,89 @@ bootstrap VOLTTRON within the virtual environment using the `--web` option:
...
+
+    [volttron]
+    message-bus = zmq
+    instance-name = my_instance
+    vip-address = tcp://127.0.0.1:22916
+    bind-web-address = https://<hostname>:8443
+    web-ssl-cert = /home/volttron/.volttron/certificates/certs/platform_web-server.crt
+    web-ssl-key = /home/volttron/.volttron/certificates/private/platform_web-server.pem
+
+The ``vctl`` tool may be used to make the changes to the $VOLTTRON_HOME/config file by choosing "Y" for the "Is this
```
[the remark this answers is not in the document]
Comment (davidraker) [reply docs/source/platform-features/web-api/introduction.rst]: Fixed. I make this typo every time to try to use it on the command line too....

File: docs/source/platform-features/web-api/introduction.rst [hunk 2]
```diff
@@ -17,6 +17,89 @@ bootstrap VOLTTRON within the virtual environment using the `--web` option:
...
+        "groups":["admin", "vui"]
+        }
+    }
+
+Users with the "vui" claim in `groups` will now be able to use the API by sending requests
+to endpoints with paths on `bind-web-address` beginning with `/vui`:
+
+::
+
+    GET https://localhost:8443/vui/
```
[the remark this answers is not in the document]
Comment (davidraker) [reply docs/source/platform-features/web-api/introduction.rst:109]: Not 100% sure I understand this comment completely, but I've made some changes to the text here to try to address it. Hopefully these address what you were looking for.

File: docs/source/platform-features/web-api/introduction.rst [hunk 3]
```diff
@@ -34,10 +127,21 @@ Available Endpoints
...
 following sections:
 
-- `Authentication <authentication-endpoints.html>`_
-- `Platforms <platform-endpoints.html>`_
-    - `Agents <agent-endpoints.html>`_
-        - `RPC <rpc-endpoints.html>`_
-    - `Devices <device-endpoints.html>`_
-    - `Historians <historian-endpoints.html>`_
-    - `Pubsub <pubsub-endpoints.html>`_
+- `Authentication <authentication-endpoints.html>`_: Endpoints for authenticating to the the API.
```
[the remark this answers is not in the document]
Comment (davidraker) [reply docs/source/platform-features/web-api/introduction.rst]: Done.

**Pull request merged**